index.html

```diff
@@ -1,229 +1,344 @@
+            <ul>
+              <li>Codekits</li>
+              <li>Codepen</li>
+              <li>GitHub</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+          <img src="./images/ic_skills.png" alt="" />
+        </div>
+        <img
+          src="./images/image_dots_with_teal.png"
+          alt="dot-matrix-image-with-teal"
+          id="dot-matrix-teal"
+        />
+      </div>
+    </section>
+    <section class="contact-section">
+      <img
+        src="./images/image_geometry_1.png"
+        alt="geometry-image-one"
+        class="geometry-image-one"
+      />
+      <img
+        src="./images/image-geometry_6.png"
+        alt="geometry-image-six"
+        id="geometry-image-six"
+      />
+      <div class="contact-form-info-container">
+        <h2 class="title-text">Get started</h2>
+        <p class="descriptive-text" id="contact-description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ex ea commodo consequat.
+        </p>
+        <form
+          action="https://formspree.io/f/mqkobewg"
+          method="post"
+          class="contact-form"
+        >
+          <div class="name-email-text-label-box">
+            <div class="input-and-text-container" id="name-label-input-id">
+              <label for="name">Your name</label>
+              <input
+                type="text"
+                name="name"
+                id="name"
+                maxlength="30"
+                required
+                placeholder="Enter your name here.."
+              />
+            </div>
+            <div class="input-and-text-container" id="email-label-input-id">
+              <label for="email">Your email</label>
+              <input
+                type="email"
+                name="email"
+                id="email"
+                required
+                placeholder="Enter your email here.."
+              />
+            </div>
+          </div>
+          <label for="message" id="message-label-id">Message</label>
+          <textarea
+            name="message"
+            id="message"
+            placeholder="Enter your message here.."
+            maxlength="500"
+            required
+          ></textarea>
+
+          <button class="primary-button">start collaboration</button>
+        </form>
+      </div>
+      <img
+        src="./images/image-geometry_7.png"
+        alt="geometry-image-seven"
+        id="geometry-image-seven"
+      />
+    </section>
+    <footer class="footer-section">
+      <a class="get-resume-btn" href="#">
+        Get my Resume <img src="./images/ic_link_teal.png" alt="" />
+      </a>
+      <div class="footer-social-links">
+        <a href="#"><img src="./images/ic_stackoverflow.png" /></a>
```

Comment:
- [ ] Kindly use a ul tag to group them, and li per icon; each icon has a link (a tag).

- Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.
- [ ] And also fix them, they are not showing.
- Figma
<img width="1440" alt="Screenshot 2023-03-03 at 03 06 06" src="https://user-images.githubusercontent.com/81444338/222709652-1fe911fa-69b8-430a-a93e-fc4d572b5e60.png">

- Your Design
<img width="1440" alt="Screenshot 2023-03-03 at 03 07 23" src="https://user-images.githubusercontent.com/81444338/222709755-02e82dfd-b31f-4532-bde7-ad1984305f18.png">